apps/airtable-cache/src/app.ts

```diff
@@ -0,0 +1,226 @@
+import { fastify, FastifyReply, FastifyRequest } from 'fastify';
```

Comment:
File is a little messy and probably too long: consider splitting into multiple files